core/DataTable.php

```diff
@@ -546,6 +557,23 @@ public function queueFilter($className, $parameters = array())
     }
 
     /**
+     * Disable a specific filter to run on this DataTable in case you have already applied this filter or if you will
+     * handle this filter manually by using a custom filter. Be aware if you disable a given filter, that filter won't
+     * be ever executed. Even if another filter calls this filter on the DataTable.
+     *
+     * @param string $className  eg 'Limit' or 'Sort'. Passing a `Closure` or an `array($class, $methodName)` is not
+     *                           supported yet. We check for exact match. So if you disable 'Limit' and
+     *                           call `->filter('Limit')` this filter won't be executed. If you call
+     *                           `->filter('Piwik\DataTable\Filter\Limit')` that filter will be executed. See it as a
+     *                           feature.
+     * @ignore
+     */
+    public function disableFilter($className)
```

Comment:
I decided not to make this method a public API yet. We can do this at any point later if we need to / want to. Also kept this feature simple as it does the job for now. If we later need to block `Piwik\DataTable\Filter\Limit` as well when `Limit` is passed we can add this any time. We never use `Piwik\DataTable\Filter\Limit` anyway